naccs_bot.py

```diff
@@ -143,6 +156,72 @@ def get_ongoing_matches(channel_id):
         return None
     return matches.json()
 
+#
+#   Get ongoing FACEIT streams for both divisions
+#
+async def get_streams():
+    # Getting our globals
+  global DISPLAYEDSTREAMS
+  global DISPLAYEDSTREAMS2
+  global toggle
+  # Resetting activestreams for each division to find so we can check to make sure they are still live
+  activestreams = {}
+  activestreams2 = {}
+  # Simple toggle / workaround. Works to reduce performance inhibitance and restricts the function to one api at a time
```

Comment:
Good thinking but it shouldn't cost us too much to hit this API once a minute or so. Let's call both endpoints at the same time every 5 minutes.

Reply:
I did this because I was having issues combining the response objects, something strange with how the dicts were being created. Ill test with combination or a simpler iteration and commit changes for that.